chain/actors/builtin/verifreg/state.go.template

```diff
@@ -170,6 +185,22 @@ func (s *state{{.v}}) GetClaims(providerIdAddr address.Address) (map[ClaimId]Cla
 {{end}}
 }
 
+func (s *state{{.v}}) GetAllClaims() (map[ClaimId]Claim, error) {
+{{if (le .v 12)}}
```

Comment:
```suggestion
{{if (le .v 8)}}
```

Reply:
Sorry, late review but after trying this out I realised this could be supported all the way back to v9, so there's no point in holding it off until we get to v13.